Makefile

```diff
@@ -3,18 +3,54 @@ format:
 	gofmt -w -e plugins scripts
 	goimports -w -e plugins scripts
 
+
 #----------------------------------------------------------------------------------
-# Retrieve GlooE build information
+# Set build variables
 #----------------------------------------------------------------------------------
-GLOOE_DIR := _glooe
-_ := $(shell mkdir -p $(GLOOE_DIR))
+# Set this variable to the name of your plugin
+PLUGIN_NAME ?= sample
+
+# Set this variable to the version of your plugin
+PLUGIN_VERSION ?= 0.0.1
 
 # Set this variable to the version of GlooE you want to target
-GLOOE_VERSION ?= 1.3.4
+GLOOE_VERSION ?= 1.3.1
+
+# Set this variable to the image name and version used for building the plugin
+GO_BUILD_IMAGE ?= golang:1.14.0-buster
```

Comment:
It is important to use the same container to build the plugin that Gloo Enterprise was built in to ensure that the same go version and linker is used during compilation. Gloo Enterprise is published alongside the container it was built in, so we have been passing that into the container and should continue to do so, rather than define it in the makefile here.

Fixing this has larger implications for the PR, which I am addressing currently

Reply:
I addressed this